content/docs/getting-started/updating-the-ui/index.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Updating the Backstage UI
+lastUpdated: '2021-08-06T12:00:00.0Z'
+description: How to configure the Backstage UI Layout
+---
+
+## Introduction
+
+Roadie allows you to configure which plugins you want to use via our UI. This guide explains the different types of
+plugin components and the different places they can be integrated into the app.
+
+## Prerequisites
+
+* You must be an admin to edit the UI layout. Please complete [this guide](/docs/getting-started/getting-started-for-admins/) before moving on to this one.
+
+## About Plugins
+
+Backstage is a plugin based application where each plugin provides a certain piece of functionality. A plugin may
+integrate with some external system such as GitHub to display information for a certain Entity e.g. Show pull request
+information for a particular service. 
+
+You can read more about Backstage plugins [here](https://backstage.io/docs/plugins/).
+
+In terms of the Backstage layout plugins can provide [three types of component](https://backstage.io/docs/plugins/composability#naming-patterns):
+1. Card - These are tiles which can be arranged into dashboards in the software catalog. E.g. The About card.
+2. Content - These are tabs which can be integrated with the catalog. E.g. The techdocs tab.
```

Comment:
TechDocs should be capitalized like it is in [the backstage.io docs](https://backstage.io/docs/features/techdocs/techdocs-overview).

```suggestion
2. Content - These are tabs which can be integrated with the catalog. E.g. The TechDocs tab.
```